blosc/frame.c

```diff
@@ -855,7 +951,16 @@ int frame_update_header(blosc2_frame* frame, blosc2_schunk* schunk, bool new) {
 
   if (frame->sdata == NULL) {
     size_t rbytes = 0;
-    FILE* fp = fopen(frame->fname, "rb");
+    FILE *fp = NULL;
+    if (frame->eframe) {
+      char* eframe_name = malloc(strlen(frame->urlpath) + strlen("/chunks.b2frame") + 1);
+      sprintf(eframe_name, "%s/chunks.b2frame", frame->urlpath);
+      fp = fopen(eframe_name, "rb+");
+      free(eframe_name);
+    }
+    else{
```

Comment:
Please fix all the `else` statements missing a space.

Reply:
Fixed in 4607ab374e512f47bc7a64117e9af8907c18fc81